frontend/src/views/Campaign.vue

```diff
@@ -626,7 +626,7 @@ export default Vue.extend({
     },
 
     messengers() {
-      return ['email', ...this.serverConfig.messengers.map((m) => m.name)];
+      return [...this.serverConfig.messengers.map((m) => m.name)];
```

Comment:
This doesn't work. Should be:

```javascript
return [...this.serverConfig.messengers];
```

Reply:
My mistake, I committed the incorrect code.  I've updated it and resubmitted